Bump github/codeql-action from 3.28.1 to 3.28.2 (#249)
rsksmart/devportal: tutorials/frontend/first-frontend-web3-injected.md

### Deploy a smart contract 

In the left side panel, go to the button Deploy and run transactions. 

> At Environment, make sure you have selected the `Injected Web3`.

Reviewer: ```suggestion
Under `Environment`, make sure you have selected the `Injected Web3` option,
as this tells Remix to use the the Web3 provider injected by a browser plugin such as MetaMask of Nifty.
```

Copy and paste the smart contract from the following gist, or copy and paste the code below:

> [https://raw.githubusercontent.com/solangegueiros/dapp-register-rsk/master/register-rsk-web3-injected/index.js](https://raw.githubusercontent.com/solangegueiros/dapp-register-rsk/master/register-rsk-web3-injected/index.js)

Reviewer: ```suggestion
[raw.githubusercontent.com/solangegueiros/dapp-register-rsk/master/register-rsk-web3-injected/index.js](https://raw.githubusercontent.com/solangegueiros/dapp-register-rsk/master/register-rsk-web3-injected/index.js)
```

Great! Now we have an information stored in our smart contract, and we can retrieve it!

> ### Note
> > We have a smart contract with a public function that has no restrictions about who is allowed to call it, but that's just for a demo.
> > 
> > In a real world smart contract there would definitely be some checks performed in any functions that alter the smart contract state.

Reviewer: ```suggestion
### Note

> We have a smart contract with a public function that has no restrictions about who is allowed to call it, but that's just for demonstration purposes.
> 
> In a real world smart contract, such checks would be performed in any functions that alter the smart contract state.
```
